src/ORM/EagerLoadedList.php

```diff
@@ -940,7 +940,7 @@ private function setDataObjectEagerLoadedData(int $id, DataObject $item): void
 
     /**
      * Gets the final rows for this list after applying all transformations.
-     * Currently only limit is applied lazily, but others could be done this was as well.
+     * Currently only limit and sort are applied lazily, but filter could be done this was as well in the future.
```

Comment:
This is an unrelated change - noticed that this PHPDoc was lying so I tidied that up while I was here.